apps/web/src/design-system/icons/general/JSLogo.tsx

```diff
@@ -0,0 +1,19 @@
+export const JSLogo = (props: React.ComponentPropsWithoutRef<'svg'>) => {
```

Comment:
```suggestion
export const JavaScriptLogo = (props: React.ComponentPropsWithoutRef<'svg'>) => {
```
Suggestion as it would make it easier to find.